XenModel/Messages.resx

```diff
@@ -6235,6 +6235,11 @@ Note: Any custom color selections will remain unchanged</value>
   <data name="GUEFI_SECURE_BOOT_MODE_LABEL" xml:space="preserve">
     <value>UEFI &amp;Secure Boot</value>
   </data>
+  <data name="GUI_NOT_COMPATIBLE" xml:space="preserve">
+    <value>This version of [XenCenter] can only connect to certain [XenServer] versions and cannot connect to {0}.

```

Comment:
I'm wondering whether we should list explicitly the versions we allow instead of saying vaguely "certain versions". What do the others think?